services/syncDonationsWithNetwork.ts

```diff
@@ -16,7 +16,7 @@ const numberOfVerifyDonationConcurrentJob =
   Number(config.get('NUMBER_OF_VERIFY_DONATION_CONCURRENT_JOB')) || 1;
 
 const cronJobTime =
-  String(config.get('VERIFY_DONATION_CRONJOB_EXPRESSION')) || '0 0 * * * *';
```

Comment:
@CarlosQ96 
I understood it's tricky because `String(undefined)` return `"undefined"`  so I changed it to `config.get('VERIFY_DONATION_CRONJOB_EXPRESSION') as string`

Reply:
Perfect I didn't know that. Thanks!